### PR TITLE
Remove redundant variable in DeltaLakeQueryRunner

### PR DESCRIPTION
**Pull request by krvikash**
## Description

Cosmetic change to delta-lake and iceberg test

## Non-technical explanation

NA

## Release notes

(X) This is not user-visible or docs only and no release notes are required.
( ) Release notes are required, please propose a release note for me.
( ) Release notes are required, with the following suggested text:

```markdown
# Section
* Fix some things. ({issue}`issuenumber`)
```

### CODE REVIEW

File: plugin/trino-iceberg/src/test/java/io/trino/plugin/iceberg/BaseIcebergConnectorSmokeTest.java [hunk 1]
```diff
@@ -51,13 +51,8 @@ protected boolean hasBehavior(TestingConnectorBehavior connectorBehavior)
         switch (connectorBehavior) {
             case SUPPORTS_TOPN_PUSHDOWN:
                 return false;
-
             case SUPPORTS_CREATE_VIEW:
-                return true;
```
[the remark this answers is not in the document]
Comment (hashhar) [reply plugin/trino-iceberg/src/test/java/io/trino/plugin/iceberg/BaseIcebergConnectorSmokeTest.java:56]: I think having related ones together is nice - much easier when skimming subclasses of BCT to see what's implemented and what's not but sadly I don't think we can enforce it - me and you don't seem to scale well @findepi.

Alternatively we can probably mention it in DEVELOPMENT.md to disable the IntelliJ codestyle which keeps reminding you to merge these together? WDYT?

File: plugin/trino-delta-lake/src/test/java/io/trino/plugin/deltalake/BaseDeltaLakeMinioConnectorTest.java [hunk 2]
```diff
@@ -102,11 +102,10 @@ protected QueryRunner createQueryRunner()
     protected boolean hasBehavior(TestingConnectorBehavior connectorBehavior)
     {
         switch (connectorBehavior) {
-            case SUPPORTS_DELETE:
```
[the remark this answers is not in the document]
Comment (findepi) [reply plugin/trino-delta-lake/src/test/java/io/trino/plugin/deltalake/BaseDeltaLakeMinioConnectorTest.java:105]: Revert --  or improve. 

For example, on current master we have a group in `BaseDeltaLakeMinioConnectorTest`

```
case SUPPORTS_DELETE:
case SUPPORTS_ROW_LEVEL_DELETE:
case SUPPORTS_UPDATE:
case SUPPORTS_NOT_NULL_CONSTRAINT:
```

the last entry in this group seems unrelated to the other three

File: plugin/trino-iceberg/src/test/java/io/trino/plugin/iceberg/BaseIcebergConnectorSmokeTest.java (hunk 1 above)
[the remark this answers is not in the document]
Comment (findepi) [reply plugin/trino-iceberg/src/test/java/io/trino/plugin/iceberg/BaseIcebergConnectorSmokeTest.java:56]: > disable the IntelliJ codestyle which keeps reminding you to merge these together?

i considered that, but it's sometimes nice on the other occasions.

we can suppress it locally for these methods

File: plugin/trino-delta-lake/src/test/java/io/trino/plugin/deltalake/BaseDeltaLakeMinioConnectorTest.java (hunk 2 above)
[the remark this answers is not in the document]
Comment (krvikash) [reply plugin/trino-delta-lake/src/test/java/io/trino/plugin/deltalake/BaseDeltaLakeMinioConnectorTest.java]: Drop the switch case change commit.

**Review by findinpath — APPROVED** [review]

**Review by ebyhr — APPROVED** [review]

**Comment by ebyhr** [comment]
Let me merge once the ongoing release is complete.

**Comment by ebyhr** [comment]
Merged, thanks!

File: plugin/trino-iceberg/src/test/java/io/trino/plugin/iceberg/BaseIcebergConnectorSmokeTest.java (hunk 1 above)
[the remark this answers is not in the document]
Comment (hashhar) [reply plugin/trino-iceberg/src/test/java/io/trino/plugin/iceberg/BaseIcebergConnectorSmokeTest.java]: I've got a PR which:
- reorders and groups all existing usages of the enums
- adds suppression where needed

Will send and request a review.
I also though of switching to switch expressions but they format very ugly for multiple labels under single case.

**Pull request merged**